### ISSUE TITLE
use java7 instead of java8 was: The app is unusable on Windows 7 because of font issues

### ISSUE THREAD

**Issue by paulvi** [agirbal/umongo#190]
I could not even connect to DB,
it is torture just to enter DB server 

![umongo-unusable](https://f.cloud.github.com/assets/1614482/1472571/29381fea-4605-11e3-8ea6-a1935db6de31.png)

**Comment by paulvi** [comment]
I could not see port 27017, it was always corrupted letters.

**Comment by agirbal** [comment]
Ok that is surprising.. Which Java version do you have installed? Have you upgraded to the latest Oracle version?
Can you try to go to tools/preferences/advanced and unclick "use native look" and then restart the app.
thanks

**Comment by paulvi** [comment]
I discovered that I have Java 8 on my PATH

> java -version
> java version "1.8.0-ea"
> Java(TM) SE Runtime Environment (build 1.8.0-ea-b109)
> Java HotSpot(TM) 64-Bit Server VM (build 25.0-b51, mixed mode)

How to specify what java version to use?

**Comment by agirbal** [comment]
If you have a terminal and a path to another jvm, you can run this line from the umongo folder:

> pathtojvm/bin/java -jar umongo.jar

I will try out with Java 8, hopefully they didnt break something again...
Btw how does it look with "use native look and feel" off?

**Comment by paulvi** [comment]
OK, now with `start-umongo.bat` like

`start "Umongo" "C:\Program Files\Java\jdk1.7.0_11\bin\java" -jar umongo.jar`

I can run and even see runtime errors in console window.

**Comment by agirbal** [comment]
so are you saying it works fine with 1.7 but text is garbled with 1.8?
that would be sad :(

**Comment by paulvi** [comment]
Yes, 1.8 spoils fonts. As 1.8 is not yet final, it may change in the future.
